fieldservice_route/views/menu.xml

```diff
@@ -0,0 +1,36 @@
+<odoo>
+
+    <!-- Routes -->
+    <menuitem id="menu_fsm_route_config"
+              name="Routes"
+              parent="fieldservice.operations"/>
+
+    <menuitem id="menu_fsm_route_dayroute"
+              name="Day Routes"
+              action="action_fsm_route_dayroute"
+              parent="menu_fsm_route_config"
+              sequence="10"
+              groups="base.group_no_one"/>
+
+    <menuitem id="menu_fsm_route"
+              name="Routes"
+              action="action_fsm_route"
+              parent="menu_fsm_route_config"
+              sequence="20"
+              groups="base.group_no_one"/>
+
+    <menuitem id="menu_fsm_route_day"
+              name="Route Days"
+              action="action_fsm_route_day"
+              parent="menu_fsm_route_config"
+              sequence="30"
+              groups="base.group_no_one"/>
```

Comment:
This one should be in Configuration and you can remove groups.

Reply:
Done